packages/types/src/clerk.ts

```diff
@@ -532,6 +532,9 @@ export type ClerkThemeOptions = DeepSnakeToCamel<DeepPartial<DisplayThemeJSON>>;
 export interface ClerkOptions {
   appearance?: Appearance;
   localization?: LocalizationResource;
+  /**
+   * @deprecated This prop has been deprecated and will be removed in the next major release.
```

Comment:
❓ is there a replacement for this prop?

Reply:
I think it will be `push` and `replace` and they will be available in v5